src/swap/types.ts

```diff
@@ -162,7 +183,7 @@ export type SwapReact = {
  * Note: exported as public Type
  */
 export type SwapError = {
-  code: number; // The error code
+  code?: number; // The error code
```

Comment:
Can we make `code` not optional? I like the idea that with every error we have a code we can track? what do you think?

Reply:
yeah i was wondering how to handle this best. we discussed changing the code to a string `SWAP_ERROR_<CODE>` which might be a good idea. the errors returned from the balance endpoints do not have a number code.

Reply:
My take here is that standardize API will take weeks, where can standardize the frontend in days.

So let's lead the way on how we think about code error strings, and than work backwards with the team to have the API reflect those as well. cc @0xAlec